prisma/schema.prisma

```diff
@@ -229,3 +231,28 @@ model DepositHead {
 
   @@map("deposit_head")
 }
+
+model MaspTransaction {
+  id               Int                 @id @default(autoincrement())
+  created_at       DateTime            @default(now()) @db.Timestamp(6)
+  updated_at       DateTime            @default(now()) @updatedAt @db.Timestamp(6)
+  transaction_hash String              @db.VarChar
+  block_hash       String              @db.VarChar
+  asset            String              @db.VarChar
+  type             MaspTransactionType
+  block_sequence   Int
+  network_version  Int
+  main             Boolean
+  amount           Int
+  event            Event?
+
+  eventId Int?
+
+  @@map("masp_transactions")
+}
+
+enum MaspTransactionType {
+  SEND
```

Comment:
What do you think about `TRANSFER`? I know we had `SEND` in the spec, but we've been using that vocabulary in the MASP prototypes